dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithP2p.ts

```diff
@@ -0,0 +1,91 @@
+import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { DataTypeEnum, type PathwaysToParisData } from "@clients/backend";
```

Comment:
Is this the content of the old P2PViewPanel test?

Reply:
It is similar. However I have thrown out test code that makes no sense or is already covered by the respective DataIntegrity test of P2p.

Reply:
(But yes, it is kind of a migration of the old P2pViewPanel test)

Reply:
ok